### PR TITLE
Highcharts license issue

### PR DESCRIPTION
**Pull request by ngehlert**
This should fix the dependency issues for highcharts that are discussed in #624 and #623

### CODE REVIEW

**Comment by ngehlert** [comment]
hm i have no idea how the git rid of the commits from the previous pull request... i tried to follow this  introduction https://help.github.com/articles/syncing-a-fork/  ... also this one https://gist.github.com/Chaser324/ce0505fbed06b947d962 but they are still there...

File: README.md [hunk 1]
```diff
@@ -20,10 +20,18 @@ Also:
 - Disabled change detection http://jsfiddle.net/gh/get/jquery/3.1.1/pablojim/highcharts-ng/tree/master/jsfiddles/disabled_change_detection/
 - Recreating chart http://jsfiddle.net/gh/get/jquery/3.1.1/pablojim/highcharts-ng/tree/master/jsfiddles/recreating/
 
+License
+-------
+This library is available under the MIT license. However it and is a wrapper for Highcharts. Highcharts itself has an own license. So make sure to follow their license as well (https://github.com/highcharts/highcharts/blob/master/license.txt)
+
+Note: You need to include the Highcharts library your self. It is not included in this repository.
```
[the remark this answers is not in the document]
Comment (pablojim) [reply README.md]: Also add a note to the setup section about needing to add highcharts or highstocks to your npm/bower etc.

**Comment by pablojim** [comment]
For clarity it's best to start a new branch from an updated master. But no worries for now.

**Comment by ngehlert** [comment]
well i did started from an updated master. those commit log entries are in my master and i'm not able to get rid of them... don't know... i will keep looking, maybe i just did something wrong. Or worst case i delete the fork and re-fork it again 😆

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ngehlert) [reply README.md]: ok, updated the setup section.

**Comment by ngehlert** [comment]
@pablojim sorry to bother you again, any updates on when you have time to verify and accept this PR? and when the next version will be released? kind of waiting for it 😄

**Comment by marcel-tov** [comment]
+1

**Comment by pablojim** [comment]
@ngehlert merged. Unfortunately I'm not going to be near my laptop until first week in December to do the release.

**Comment by ngehlert** [comment]
@pablojim ok thanks for your response :) looking forward to first week in december then 😄

**Comment by pablojim** [comment]
@ngehlert released now!

**Comment by ngehlert** [comment]
@pablojim awesome stuff. thanks

**Pull request merged**